.github/workflows/container.yml

```diff
@@ -0,0 +1,57 @@
+name: container
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+      - rel/*
+      - feature/*
```

Comment:
do we need to auto-build it on every feature branch push? i didn't see the ticket mention that

Reply:
you have the manual trigger for building on this branch. I'd add alphanet specifically if we need it too.

Reply:
this is non-blocking. you can merge. just considering that we don't know how many minutes github gives us for free, i thought we'd err on the minimal side for now.